src/main/java/com/ocelotslovebirds/birdhaus/blocks/BirdhouseBlockEntity.java

```diff
@@ -180,8 +184,25 @@ private void removeBirdHouseGoalToSurroundingParrots() {
     private void handleSeedsForTick() {
         // Tick the birdhouse to check it needs to consume a seed
         if (tickerForSeedConsumption.tick()) {
+            ItemStack seeds = itemHandler.extractItem(0, 1, false);
+
+            if (seeds.is(Tags.Items.SEEDS_WHEAT)){
+                birdColor = 3;
+            }
+            else if (seeds.is(Tags.Items.SEEDS_BEETROOT)){
+                birdColor = 1;
+            }
+            else if (seeds.is(Tags.Items.SEEDS_MELON)){
+                birdColor = 2;
+            }
+            else if (seeds.is(Tags.Items.SEEDS_PUMPKIN)){
+                birdColor = 4;
+            }
+            else {
+                birdColor = 0;
+            }
             // If a seed was successfully extracted from the birdhouse
-            if (!itemHandler.extractItem(0, 1, false).isEmpty()) {
+            if (seeds.isEmpty()) {
```

Comment:
You changed the behaviour here, was it intentional? Previously it checked if the seeds were not empty

Reply:
That's a typo! Good catch, I'll change it when I get home

Reply:
done